jwst_scripts/openspace-jwst-buttons.js

```diff
@@ -0,0 +1,23 @@
+//helper function for nicely fading groups of trails
+var showTrails = (objects) => {
```

Comment:
Where is this function being used?  Looks like the strings contained in `objects` are expected to a have a . at the end:
`openspace.getPropertyValue("Scene." + object + "Trail.Renderable.Enabled"`  Maybe would be better to not require that?

Reply:
Since I don't have any JWST buttons maybe it is beter to remove this file completely since it is never used.